src/display/text_layer.js

```diff
@@ -166,6 +166,8 @@ function appendText(task, geom, styles, ctx) {
   textDiv.style.fontSize = `${fontHeight}px`;
   textDiv.style.fontFamily = style.fontFamily;
 
+  textDivProperties.fontSize = fontHeight;
```

Comment:
Please add a default value for the `fontSize`-property in the object definitions above, see https://github.com/mozilla/pdf.js/blob/8ebd2d3dfd0c32086314014f8006c91913c2f746/src/display/text_layer.js#L123-L141